src/KMeansResult.ts

```diff
@@ -39,10 +51,10 @@ export default class KMeansResult {
    * @param {Array<Array<number>>} data - the [x,y,z,...] points to cluster
    * @return {KMeansResult}
    */
-  computeInformation(data) {
-    let enrichedCentroids = this.centroids.map((centroid) => {
+  computeInformation(data: Array<Array<number>>): KMeansResult {
```

Comment:
With simple types, I find the other syntax for arrays easier to read
```suggestion
  computeInformation(data: number[][]): KMeansResult {
```